app/src/main/java/com/example/iphonesearchapi/fragments/DetailsFragment.kt

```diff
@@ -5,15 +5,47 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import com.example.iphonesearchapi.R
+import androidx.appcompat.app.ActionBar
+import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.fragment.navArgs
+import com.example.iphonesearchapi.databinding.FragmentDetailBinding
+import com.example.iphonesearchapi.model.ResultOf
+import com.example.iphonesearchapi.utility.showLoadingIndicator
+import com.example.iphonesearchapi.viewmodel.detailViewModel
+import org.koin.androidx.viewmodel.ext.android.viewModel
 
-class BlankFragment : Fragment() {
+class DetailsFragment : Fragment() {
+
+    private val viewModel: detailViewModel by viewModel()
+    private lateinit var binding : FragmentDetailBinding
 
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        return super.onCreateView(inflater, container, savedInstanceState)
+
+        binding = FragmentDetailBinding.inflate(layoutInflater)
+        val args: DetailsFragmentArgs by navArgs()
+        val actionbar: ActionBar =  (activity as AppCompatActivity).supportActionBar!!
+
+
+        actionbar.title =args.songName
```

Comment:
If you only need to set the title on action bar, you can try setting the title on activity directly
```kotlin
requireActivity().title = args.songName
```